src/components/Editor/Reducers/loadProjectReducers.test.js

```diff
@@ -0,0 +1,257 @@
+      accessDeniedNoAuthModalShowing: undefined,
+      notFoundModalShowing: true,
+      currentLoadingRequestId: undefined,
+    };
+    expect(reducer(initialState, loadRejectedAction)).toEqual(expectedState);
+  });
+
+  describe("When it errors with an auth token", () => {
+    let expectedState;
+
+    beforeEach(() => {
+      loadRejectedAction.meta.arg.accessToken = "I exist";
+
+      expectedState = {
+        loading: "failed",
+        saving: "idle",
+        accessDeniedWithAuthModalShowing: true,
+        accessDeniedNoAuthModalShowing: undefined,
+        notFoundModalShowing: undefined,
+        currentLoadingRequestId: undefined,
+      };
+    });
+
+    test("If the error is a 500 and there is a token, it sets up state accordingly", () => {
+      loadRejectedAction.error.message = "Request failed with status code 500";
+      expect(reducer(initialState, loadRejectedAction)).toEqual(expectedState);
+    });
+
+    test("If the error is a 403 and there is a token, itsets up state accordingly", () => {
+      loadRejectedAction.error.message = "Request failed with status code 403";
+      expect(reducer(initialState, loadRejectedAction)).toEqual(expectedState);
+    });
+
+    test("If the error is any other status code and there is a token, it sets up state accordingly", () => {
+      loadRejectedAction.error.message = "Request failed with status code 404";
+      expect(reducer(initialState, loadRejectedAction)).not.toEqual(
+        expectedState,
+      );
+    });
+  });
+
+  describe("When it errors without an auth token", () => {
+    let expectedState;
+
+    beforeEach(() => {
+      initialState = {
+        ...initialState,
+        modals: {
+          accessDenied: {},
+        },
+      };
+
+      expectedState = {
+        loading: "failed",
+        saving: "idle",
+        accessDeniedWithAuthModalShowing: undefined,
+        accessDeniedNoAuthModalShowing: true,
+        notFoundModalShowing: undefined,
+        currentLoadingRequestId: undefined,
+        modals: {
+          accessDenied: {
+            identifier: "I am he",
+            projectType: "Err",
+          },
+        },
+      };
+    });
+
+    test("If the error is a 500 and there is a token, it sets up state accordingly", () => {
+      loadRejectedAction.error.message = "Request failed with status code 500";
+      expect(reducer(initialState, loadRejectedAction)).toEqual(expectedState);
+    });
+
+    test("If the error is a 403 and there is a token, it sets up state accordingly", () => {
+      loadRejectedAction.error.message = "Request failed with status code 403";
+      expect(reducer(initialState, loadRejectedAction)).toEqual(expectedState);
+    });
+
+    test("If the error is a 401 and there is a token, it sets up state accordingly", () => {
+      loadRejectedAction.error.message = "Request failed with status code 401";
+      expect(reducer(initialState, loadRejectedAction)).toEqual(expectedState);
+    });
+  });
+});
```

Comment:
💪 nice additions to the tests. Like you said previously the `reducers` are simple enough to test and this approach works, the new alt method now gives us access to test `extraReducers` 🙌